resources/views/components/docs/sidebar.blade.php

```diff
@@ -7,7 +7,7 @@
 	x-transition:leave-start="top-0"
 	x-transition:leave-end="-top-full">
 
-	<div class="lg:sticky lg:w-56 xl:w-64 lg:px-0 lg:top-0 overflow-y-auto bg-white lg:h-screen fixed left-0 top-nav z-50 overflow-x-auto text-left px-4 h-screen-24 w-full"
+	<div class="lg:sticky lg:w-56 xl:w-64 lg:px-0 lg:top-0 overflow-y-auto bg-white dark:bg-gray-900 lg:h-screen fixed left-0 top-nav z-50 overflow-x-auto text-left px-4 h-screen-24 w-full"
```

Comment:
Add `scrollbar` utility class to have the customized scrollbar.

Reply:
Okay, I will Do it

Reply:
But what's is the issue to make it for the main scrollbar too? 🤔

Reply:
It's not a issue but in most cases it's good to keep the default browser scrollbar on the outer page and just customize the scrollbar in individual components

Reply:
Okay

Reply:
Done